src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -466,6 +466,15 @@ public string EvalDependentExpression(string expr)
                     {
                         DisplayHint = results.FindString("displayhint");
                     }
+                    if (results.Contains("attributes"))
+                    {
+                        _isReadonly = false;
```

Comment:
this is redundant. _isReadOnly = false, can be moved to the constructor or class initializer. 
I see a similar pattern in line 238